village/src/App.js

```diff
@@ -27,10 +28,20 @@ class App extends Component {
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
-      <div className="App">
-        <SmurfForm updateItems={this.updateItems} />
-        <Smurfs smurfs={this.state.smurfs} />
-      </div>
+      <BrowserRouter>
+        <div className="App">
+          <ul>
```

Comment:
nav links, was supposed to be in its own component, but im positive you can easily do this